assets/app/features/upgrade/templates/upgrade7/openstack-services.jade

```diff
@@ -0,0 +1,10 @@
+h2(translate='') upgrade7.steps.openstack-services.title
+p(translate='') upgrade7.steps.openstack-services.description
+
+button.btn.btn-success.center-block.btn-lg(translate='', 
```

Comment:
the specification calls for a spinner on this button. use `<suse-lazy-spinner>` directive if possible.